src/nft/components/card/containers.tsx

```diff
@@ -0,0 +1,849 @@
+import { t, Trans } from '@lingui/macro'
```

Comment:
this file is large and should be split up. A file for icons, for util functions (add some of the util functions from index for consistency?), action button, etc